UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs

```diff
@@ -122,7 +113,7 @@ private void NodeScatter(Vector3Int location, OreProbability materialSpecified)
 			var chosenLocation = locations[RANDOM.Next(locations.Count)];
 			var ranLocation = chosenLocation + DIRECTIONS[RANDOM.Next(DIRECTIONS.Count)];
 			var tile = metaTileMap.GetTile(ranLocation);
-			if (tile != null && tile.name.Contains("rock_wall"))
+			if (tile != null)
```

Comment:
A tile adjacent to a mineable tile might not be mineable, so you'll need to check that the location exists in mineableTiles.